build.sbt

```diff
@@ -3,11 +3,11 @@ import play.sbt.PlayImport.PlayKeys.playDefaultPort
 import uk.gov.hmrc.DefaultBuildSettings
 import uk.gov.hmrc.DefaultBuildSettings.*
 
-ThisBuild / scalaVersion := "2.13.16"
+ThisBuild / scalaVersion := "3.7.3"
```

Comment:
I appreciate the ticket was raised back in June so there's a discrepancy here. Is 3.7.3 considered the latest stable version?